src/ood-gen/bin/gen.ml

```diff
@@ -1,4 +1,4 @@
-open Cmdliner
+open Cmdliner 
```

Comment:
Looks like there's a whitespace change here which is probably indicative of the code needing to be formatted using `ocamlformat`. You can do that by calling `make fmt` (this is the reason the build is currently failing, we check to make sure the code is formatted before trying to do any building or testing).